proc/redis/session.go

```diff
@@ -108,6 +108,10 @@ func (s *session) loopWrite() {
 		req.Wait()
 		// TODO(kirk91): abstract response
 		resp := req.Response()
+		// asking request is added by sam during slot migration, thus resp should be swallowed
+		if req.body.Equal(askingRequest) {
+			continue
+		}
```

Comment:
A bit weird... The `askingRequest` shouldn't be inside `processingReqs`.

Reply:
yes! the req will be some kind of `get`,  may be use a filter will be a better solution?

Reply:
I mean, as session and client(upstream) have their own `processingReqs`, the ASKING request created by upstream should just be inside `client.processingReqs`, not the one in session. 
And I can't find any codes that will pass ASKING request to from `client.processingReqs` to `session.processingReqs`.

Reply:
> I mean, as session and client(upstream) have their own `processingReqs`, the ASKING request created by upstream should just be inside `client.processingReqs`, not the one in session. 

yeah, i got it

> And I can't find any codes that will pass ASKING request to from `client.processingReqs` to `session.processingReqs`.

it's done in the hook, the hook will be called when upstream request finished, like below:

```go
	simpleReq.RegisterHook(func(simpleReq *simpleRequest) {
		req.SetResponse(simpleReq.Response())
	})
```

Reply:
```go
		askingReq := newSimpleRequest(newArray(
			*newBulkString(ASKING),
		))
		u.MakeRequestToHost(hostAddr, askingReq)
		u.MakeRequestToHost(hostAddr, req)
```

So I took a closer look seems no bug for the opensource version, I'll try to run the integration test local to make sure. And may be eleme's code version is in different logic.

Reply:
The test passed, So we just need to update the test